content/index.id.md

```diff
@@ -99,79 +99,79 @@ Sarana penulisan adalah perangkat lunak dan jasa yang digunakan oleh "para penul
 
 Informasi seputar ATAG:
 - [Gambaran Umum ATAG](/standards-guidelines/atag/)
-- [[Selayang Pandang ATAG]](/standards-guidelines/atag/glance/)
+- [[Sekilas Mengenai ATAG]](/standards-guidelines/atag/glance/)
 - [Standar ATAG 2.0](https://www.w3.org/TR/ATAG/)
 
 ### Pedoman Aksesibilitas Agen Pengguna (UAAG) {#uaag}
 
-Agen pengguna termasuk browser, ekstensi browser, pemutar media, perangkat pembaca, dan aplikasi lainnya yang menerjemahkan konten web.
+Agen pengguna termasuk browser, ekstensi browser, pemutar media, perangkat pembaca, dan aplikasi lainnya yang menyajikan konten web.
 
 Informasi seputar UUAG:
 - [Gambaran Umum UAAG](/standards-guidelines/uaag/)
 - [Catatan UAAG 2.0](https://www.w3.org/TR/UAAG20/)
 
-### Draf Kerja Pedoman Aksesibilitas Konten Web (WCAG) 3 W3C {#wcag3}
+### Rancangan Pedoman Aksesibilitas Konten Web (WCAG) 3 W3C {#wcag3}
 
-WCAG 3 adalah draf awal yang ditujukan untuk menjadi Standar W3C. WCAG 3 berlaku untuk konten web, aplikasi, alat, penerbitan, dan teknologi baru dalam web.
+WCAG 3 adalah rancangan awal yang ditujukan untuk menjadi Standar W3C. WCAG 3 berlaku untuk konten web, aplikasi, alat, penerbitan, dan teknologi baru dalam web.
 
 Informasi seputar WCAG 3:
-- [[Pengantar WCAG 3]](/standards-guidelines/wcag/wcag3-intro/)
+- [[Pengenalan ke WCAG 3]](/standards-guidelines/wcag/wcag3-intro/)
 
 ## Spesifikasi Teknis
 
-### Aplikasi Internet Kaya yang Aksesibel (WAI-ARIA) {#aria}
+### Aplikasi Internet yang Kaya dan Aksesibel (WAI-ARIA) {#aria}
```

Comment:
@frirasyidi Would you please guide me through the rationale behind this change?

Reply:
Yes. The second sentence makes it clearer that the one that is rich is the "internet application" instead of the "internet".